src/utils/extract.tsx

```diff
@@ -0,0 +1,12 @@
+class Extract {
```

Comment:
This is fine, but I wonder why this couldn't be just a function `extractUuid`?

Reply:
✅